task_3/conversion.js

```diff
@@ -0,0 +1,35 @@
+class Converter {
+    constructor (input){
+        this.input = input
+    }
+
+    stringToNumber (input) {
+        console.log(Number(input));
+    }
+
+    numberToString (input) {
+        console.log(String(input))
+    }
+
+    datesToNumber (input) {
+        console.log(Number(input))
+    }
+
+    dateToString (input) {
```

Comment:
This function is the same as numberToString(), in pure js there is no strong typing, so numberToString and dateToString are the same method with different names.